h2o-extensions/xgboost/src/test/java/hex/tree/xgboost/DefaultMojoImplTest.java

```diff
@@ -91,4 +114,27 @@ private static XGBoostModel trainModel(Booster booster) {
     return model;
   }
 
+  /**
+   * Old mojo writer that does not include use_java_scoring_by_default
+   */
+  private static final class OldSerializationXGBMojoWriter extends XGBoostMojoWriter {
```

Comment:
can we just commit a small old mojo in the repository instead of generating it like this?

Reply:
I kinda like this approach a little more because the difference in mojo is visible from code

Reply:
I also prefer to add old mojo in the repository. The difference is visible from code; however, when someone adds some new parameter, and he/she would like to test it, he/she would have to implement a new object where only one parameter will be different, etc...